src/main/java/frc/robot/RobotContainer.java

```diff
@@ -96,6 +101,18 @@ private void controllerBindings() {
       // Set the drive powers based on which controller is being used
       if (isXbox) {
         tankSubsystem.setCarDrivePowers(-controlXbox.getY(Hand.kLeft), controlXbox.getX(Hand.kRight));
+        if(controlXbox.getAButtonPressed()){
+          clawSubsystem.openClaw();
+        }
+        if(controlXbox.getBButtonPressed()){
+          clawSubsystem.openClaw();

```

Comment:
Let's use toggles -- pressing a button closes if it's open, or opens if it closes. Will make it much easier for the drivers. We should also have a second xbox controller for the second driver.